src/operator/numpy/np_elemwise_broadcast_op_extended.cc

```diff
@@ -371,5 +371,71 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_npi_rldexp_scalar)
 .set_attr_parser([](NodeAttrs *attrs) { attrs->parsed = std::stod(attrs->dict["scalar"]); })
 .set_attr<FCompute>("FCompute<cpu>", BinaryScalarOp::Backward<cpu, mshadow_op::rldexp_grad>);
 
+MXNET_OPERATOR_REGISTER_BINARY_BROADCAST(broadcast_fmax)
+.add_alias("_npi_fmax")
+.set_attr<FCompute>("FCompute<cpu>", BinaryBroadcastCompute<cpu, mshadow_op::fmax>)
+.set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseIn{"_backward_broadcast_fmax"});
+
+NNVM_REGISTER_OP(_backward_broadcast_fmax)
```

Comment:
simply use `_backward_npi_fmax`, and get rid of the alias, similar for other ops.